velox/docs/functions/presto/tdigest.rst

```diff
@@ -0,0 +1,134 @@
+depending on the compression factor used. In comparison, quantile digest supports more
+numeric types and provides a maximum rank error guarantee, which ensures relative uniformity
+of precision along the quantiles. Quantile digests are also formally proven to support
+lossless merges, while T-digest is not (though it does empirically demonstrate lossless merges).
+
+T-digest was developed by Ted Dunning and is more restrictive in its type support,
+accepting only ``double`` type parameters. This contrasts with quantile digest, which
+supports a broader range of numeric types including ``bigint``, ``double``, and ``real``,
+making quantile digest more versatile for different data types.
+
+Velox uses the modern KLL sketch algorithm for the ``approx_percentile`` function, which
+provides stronger accuracy guarantees than both T-digest and quantile digest.
+The T-digest functions documented here exist primarily to support
+pre-existing workloads that have data stored using the `T-digest
+<https://doi.org/10.1016/j.simpa.2020.100049>`_ format for backward compatibility.
+
+Data Structures
+---------------
+
+A T-digest is a data sketch which stores approximate percentile information.
+The Velox type for this data structure is called ``tdigest``,
+and it accepts a parameter of type ``double`` which represents the set of
+numbers to be ingested by the ``tdigest``.
+
+T-digests may be merged without losing precision, and for storage and retrieval
+they may be cast to/from ``VARBINARY``.
+
+Functions
+---------
+
+Aggregate Functions
+^^^^^^^^^^^^^^^^^^^
+
+.. function:: tdigest_agg(x) -> tdigest<double>
+
+    Returns the ``tdigest`` which summarizes the approximate distribution of all input values of ``x``.
+    ``x`` must be of type ``double``.
+    The default compression factor is ``100``.
+
+.. function:: tdigest_agg(x, w) -> tdigest<double>
+   :noindex:
+
+    Returns the ``tdigest`` which summarizes the approximate distribution of all input values of ``x`` using per-item weight ``w``.
+    ``x`` must be of type ``double``. ``w`` must be of type ``double``.
+    The default compression factor is ``100``.
+
+.. function:: tdigest_agg(x, w, compression) -> tdigest<double>
+   :noindex:
+
+    Returns the ``tdigest`` which summarizes the approximate distribution of all input values of ``x`` using per-item weight ``w`` and the specified compression factor.
+    ``x`` must be of type ``double``. ``w`` must be of type ``double``. ``compression`` must be of type ``double``.
+    ``compression`` must be a positive constant for all input rows. The default is ``100``, maximum is ``1000``, and values lower than ``10`` are rounded to ``10``. Higher compression means more accuracy at the cost of more memory.
+
+.. function:: merge(tdigest<double>) -> tdigest<double>
+    :noindex:
+
+    Merges all input ``tdigest``\ s into a single ``tdigest``.
+
+Scalar Functions
+^^^^^^^^^^^^^^^^
+
+.. function:: value_at_quantile(digest, quantile) -> double
```

Comment:
You could use Types in the function string description itself instead of adding text. Other documentation in Velox uses format's like

ST_Touches(geometry1: Geometry, geometry2: Geometry) -> boolean

Multiple function strings can be specified for different types.

Reply:
Updated this.